### DISCUSSION TITLE
Possibility of adding a highway

### DISCUSSION THREAD

**Discussion by ravenfeld** [Helium314/SCEE#562] [Ideas]
I don't know if it's technically possible, I'm not quite mature enough with the project yet.

In my case, I often have to wait until I get home to design little bits of road, 
I know that Vespucci does it but I'm having a lot of trouble with it in my case.

I'm thinking that with SC's pretty interface it might be nice.

**Comment by HolgerJeromin** [comment]
You can (with SC and SCEE) create a gps log, upload to osm.org and attach the path to a note.
Creating and connecting to other ways is beyond both apps.

**Reply by ravenfeld** [reply]
Yes, that's what I'm doing to contribute directly without going through JOSM again. But I understand that at some point the application must not be able to do everything to remain understandable.

**Comment by Helium314** [comment]
Technically creating new ways is possible; splitting a way actually creates new ways.
For creating ways as you probably want, the main thing necessary is a good(!) UI, but also an bunch of checks will be necessary.
